sdk/servicebus/service-bus/package.json

```diff
@@ -102,11 +116,12 @@
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
     "preinstall": "node ../../../common/scripts/rush-welcome.js",
+    "pretest": "npm run build-test",
```

Comment:
`pretest` should be removed, `test` and its subcommands already run `build:test` explicitly

Reply:
Got it, I've updated this along with other scripts per comments @bsiegel